sage_acsv/kronecker.py

```diff
@@ -65,6 +66,9 @@ def _kronecker_representation(system, u_, vs, lambda_=None, linear_form=None):
         ideal = MPolynomialIdeal(rabinowitsch_R, ideal.groebner_basis())
     except Exception:
         raise ACSVException("Trouble computing Groebner basis. System may be too large.")
+    
+    if ideal.dimension() != 0:
+        raise ACSVException("Ideal is not 0-dimensional - cannot compute kronecker representation.")
```

Comment:
Including the ideal might help every once in a while:

```suggestion
        raise ACSVException(f"Ideal {ideal} is not 0-dimensional. Cannot compute Kronecker representation.")
```